### ISSUE TITLE
Problem with paths on Windows

### ISSUE THREAD

**Issue by ChrislyBear-GH** [brandoncc/rails-project-navigator#1]
When I try to navigate to any file I get this error:
```
Couldn't open the file. Error: Error: cannot open file://c:\users\xxx\documents\coding\rails_app\app\controllers\comments_controller.rb. Details: Invalid File Resource (file://c:\users\...
```

As you can see, the slashes are wrong. They need to be forward slashes to be a valid file source. Also, the third forward slash after "file://" is missing.

See here for the correct syntax on Windows: https://en.wikipedia.org/wiki/File_URI_scheme#Windows

Could you please look into this, Brandon? It would be very much appreciated!

All the best,
Chris

**Comment by brandoncc** [comment]
Hi Chris,

Thanks for reporting this. I have never used seeing_is_believing on windows so I didn't realize it worked and I needed to develop the extension with that in mind. 

I'm currently under the weather but I will work on this sometime this week. 

Thanks again for your feedback,
Brandon

**Comment by ChrislyBear-GH** [comment]
Hey Brandon,

Thanks for your quick response! No need to hurry, just wanted to make you aware of the issue.

Get well soon!
Chris

**Comment by brandoncc** [comment]
Oops, as you can see I was mistaken about which extension you were referring to (I mentioned `seeing_is_believing`). I'm looking into this now though 👍

**Comment by brandoncc** [comment]
Hi Chris,

I have addressed the problem, but I am having a problem publishing the latest version. As soon as I figure out what the problem with publishing is, I will publish it.

I also identified another bug while working on this and fixed it, and updated the tests so they will pass on Windows now as well. This has definitely taught me something about developing extensions for multiple platforms, so thank you for that!
